interop/deposit_contract_testnets/README.md

```diff
@@ -0,0 +1,69 @@
+# Guidelines for Eth2 testnets using a deposit contract
+
+This document provides a set of implementation guidelines for clients, aiming to simplify the interoperability testing in Eth2 phase 0 testnets using an Eth1 validator deposit contract.
+
+## Client-operated testnets
+
+An Eth2 testnet can be identified by an address of a deployed validator deposit contract, running on the Görli network. Multiple testnets can be running at the same time and all client teams are encouraged to deploy as many contracts as they see fit. 
+
+To facilitate discovering and connecting to existing testnets, all client teams are encouraged to publish metadata files for their active testnets in the following repository:
+
+https://github.com/eth2-clients/eth2-testnets
+
+Please create a sub-folder for your team, where each active testnet will be stored as a nested sub-folder with a descriptive name. For example: 
+
+```
+trinity/1000_validators_testnet/bootstrap_nodes.txt
+trinity/1000_validators_testnet/deposit_contract.txt
+trinity/1000_validators_testnet/config.yaml
+trinity/1000_validators_testnet/genesis.ssz
+```
+
+`bootstrap_nodes.txt` is a line-delimited text file with [multiaddr](https://github.com/multiformats/multiaddr) records for the bootstrap nodes of the testnet. For example:
+
+```
+/ip4/10.20.30.40/tcp/9100/p2p/16Uiu2HAmEAmp4FdpPzypKwTMmsbCdnUafDvXZCpFrUDbYJZNk7hX
+/ip4/10.20.30.50/tcp/9100/p2p/16Uiu2HAmV5SpcmrEymmaj1mymSHkbMpMVUXFRiXRxpHkfWdQ7bi3
+```
+
+To connect to the testnet, the user should either pass the entire file as a command-line parameter named `bootstrap_file` or she should specify the individual entries with a repeatable command-line parameter named `bootstrap_node`.
+
+`deposit_contract.txt` is a single-line text file with the address of the testnet's deposit contract on the Görli network. To connect to the testnet, the user should specify the address as a command-line parameter named `deposit_contract`.
```

Comment:
The deposit contract address is already included in `config.yaml`. Perhaps this is duplication?

Reply:
Long-term, it makes a lot of sense for the deposit contract to be considered a constant and thus, part of the preset, but right now we'll be switching between testnets quite frequently and I think having a command-line parameter would be quite helpful. We can consider the address in the preset as the default value for this parameter.

Reply:
Sure. Does it mean that the address in `config.yaml` should be ignored if this parameter is provided?

Reply:
Yes, I've tried to edit the text a bit to clarify this.